src/services/onboardingService.ts

```diff
@@ -0,0 +1,71 @@
+import {getFirstKeyWithRelation} from '@sphereon/ssi-sdk-ext.did-utils';
```

Comment:
does it make sense to give this file a name that mentions it is a service for the onboarding machine? I think it would be good to make a difference between "normal" services, and the machine services

Reply:
maybe also for the GlobalServices?

Reply:
Not sure why. The service has no knowledge of state machines at all. It has a simple interface for it's argument only containing data and it has a simple result, completely unrelated to the machine.

I can quite easily call this service with some personalData standalone. It's inputs and outputs are very similar to any other service.

We can rename the GlobalService to something like GlobalMachineServices, but the same thing should apply there. Given the decoupling we are creating now the services should have no knowledge of being part of a machine to begin with.